src/pages/Access.tsx

```diff
@@ -36,17 +37,40 @@ function extractSwarmHash(string: string): string | null {
   return (matches && matches[0]) || null
 }
 
-function recognizeSwarmHash(value: string) {
-  if (value.length < 64) {
-    return value
+function extractSwarmCid(s: string): string | null {
```

Comment:
Why have you decided to use `null`? Wouldn't simple `undefined` be enough? I would generally suggest avoiding `null` as much as possible...

Reply:
I copied the code from the dashboard where it was implemented like this. I agree with not using null, it would be best enforced with an eslint rule.